src/generators.rs

```diff
@@ -1715,6 +1715,62 @@ pub fn dorogovtsev_goltsev_mendes_graph(py: Python, n: usize) -> PyResult<graph:
     })
 }
 
+/// Generates Zachary's Karate Club graph.
+///
+/// Zachary's Karate Club graph is a well-known social network that represents
+/// the relations between 34 members of a karate club, as described in the
+/// paper by Zachary [1]_.
+///
+/// The graph is undirected and contains 34 nodes (members) and 78 edges
+/// (interactions). Each node represents a club member, and each edge represents
+/// a relationship between two members.
```

Comment:
Maybe we should mention what the node and edge data payloads are since they're fixed in the output.

Reply:
Done. Hopefully it makes sense, I wanted to give callers the freedom to choose their types but the original paper is pretty fixed about what the edge weights are.